ios/VizbeeCastButtonView.m

```diff
@@ -43,6 +56,8 @@ - (void)layoutSubviews {
     if(nil != self.tintColor){
         [self setTintColor:self.tintColor];
     }
+    [self.castButton setEnabled:self.enabled];
```

Comment:
The default value of `enabled` is false and for backward compatibility, this will always be set to false